shared/js/background/tracker-utils.js

```diff
@@ -74,6 +74,7 @@ function truncateReferrer (referrer, target) {
     } else {
         modifiedReferrer = utils.extractLimitedDomainFromURL(referrer, {keepSubdomains: true})
     }
+    modifiedReferrer += '/'
```

Comment:
I think we should move this to `utils.extractLimitedDomainFromURL` (specifically to `${parsedURL.protocol}//${finalURL}**/**` part)? We are using this function only here so there is no risk of breaking anything else and appending something to URL here seems a bit out of place.

Reply:
yes great thinking, thanks!